### ISSUE TITLE
Fix 64-bit Windows support

### ISSUE THREAD

**Issue by pmachapman** [pmachapman/helpdeco#8]
The 64-bit build is buggy and broken for Visual Studio on Windows.

**Comment by sskras** [comment]
Is this not fixed yet? Asking per https://github.com/pmachapman/helpdeco/commit/82aaa87593bf2b50ea25b5624bd82155aa12a13d#diff-874e1ab6231f76437898785747235e071c862a8d48ce8244634c4109a872962bR16 :)

**Comment by pmachapman** [comment]
No, I haven't looked at the bugs on x64. I only added the build, realised it was broken, then created this issue. The 32-bit build is fine AFAIK.
